lighthouse-core/fraggle-rock/gather/navigation-runner.js

```diff
@@ -48,14 +46,11 @@ const DEFAULT_HOSTNAME = '127.0.0.1';
 const DEFAULT_PORT = 9222;
 
 /**
- * @param {{driver: Driver, config: LH.Config.FRConfig, options?: InternalOptions}} args
+ * @param {{driver: Driver, config: LH.Config.FRConfig}} args
  * @return {Promise<{baseArtifacts: LH.FRBaseArtifacts}>}
  */
-async function _setup({driver, config, options}) {
+async function _setup({driver, config}) {
   await driver.connect();
-  if (!options?.skipAboutBlank) {
-    await gotoURL(driver, defaultNavigationConfig.blankPage, {waitUntil: ['navigated']});
```

Comment:
I'm thinking I'll bring this back just in case, but why do we have this extra `about:blank` navigation at the start in the first place? There was already an `about:blank` navigation at the start of each config nav in `_setupNavigation`